### PR TITLE
Translation for pt_br

### PR DESCRIPTION
**Pull request by thiagokenis**
File containing pt_br translation.

### CODE REVIEW

File: src/main/resources/assets/modmenu/lang/pt_br.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "modmenu.title": "Mods",
+  "modmenu.loaded": "(%s Carregado)",
```
Comment (creeper123123321) [inline src/main/resources/assets/modmenu/lang/pt_br.json]: Plural

Comment (Prospector) [reply src/main/resources/assets/modmenu/lang/pt_br.json]: What should it be?

Comment (thiagokenis) [inline src/main/resources/assets/modmenu/lang/pt_br.json]: {
  "modmenu.title": "Mods",
  "modmenu.loaded": "(%s Carregados)",

Comment (Prospector) [reply src/main/resources/assets/modmenu/lang/pt_br.json]: Fixed

**Pull request merged**